scripts/Dockerfile.build

```diff
@@ -22,6 +22,8 @@ RUN apk update && \
     chmod +x /usr/local/bin/terraform && \
     rm -rf tmp
 
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /usr/local/bin v1.53.3
```

Comment:
```suggestion
```
actions will already run golangci-liint, install on docker is not needed

Reply:
added there to have available running here locally with docker

Reply:
In that case does it makes sense to have separate dockerfile for local and CI?